Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,585 @@
+
+Start by cloning the `catena-tools-core` repo:
+
+**Git Bash**
+```bash
+git clone git@github.com:CatenaTools/catena-tools-core.git
+cd catena-tools-core/
+```
+
+Then add a new remote and do a git push, this will deploy the `HEAD` of the repo. The command to do so is the value of the `add_doku_remote_command` output. The push command is platform specific, use the `{powershell,unix,windows}_deploy_command` output above.
+
+**Git Bash**
+```bash
+git remote add dokku dokku@dev.catenatools.com:platform
+git push dokku HEAD:main ## Replace this with your platform's deploy command
+```
+
+Dokku will then show deploy logs:
+
+```
+remote: Resolving deltas: 100% (5066/5066), done.
+-----> Set main as deploy-branch
+-----> Cleaning up...
+-----> Building platform from Dockerfile
+remote: #0 building with "default" instance using docker driver
+remote:
+remote: #1 [internal] load build definition from Dockerfile
+remote: #1 transferring dockerfile: 1.50kB done
+remote: #1 DONE 0.1s
+remote:
+remote: #2 [internal] load metadata for mcr.microsoft.com/dotnet/sdk:8.0
+remote: #2 DONE 0.2s
+remote:
+remote: #3 [internal] load metadata for docker.io/library/debian:latest
+remote: #3 DONE 0.4s
+remote:
+remote: #4 [internal] load .dockerignore
+remote: #4 transferring context: 51B done
+remote: #4 DONE 0.0s
+
+### Truncated for Brevity
+
+=====> Application deployed:
+       http://platform.dev.catenatools.com
+       https://platform.dev.catenatools.com
+
+To jlyon1.catenatools.com:platform
+ * [new branch]      HEAD -> main
+```
+
+Now we're all set! 
+
+> You can update your platform the same way, simply commit the changes and push them to the `main` branch on the dokku remote. The server will automatically build and deploy your changes with no downtime.
+
+
+You can test your platform by logging in:
+
+**Git Bash**
+```Bash
+SESSION_ID=$(curl -D - -sS --location '$CATENA_URL/api/v1/authentication/login' \
+      --header 'Content-Type: application/json' \
+      --data '{
+      "provider": "PROVIDER_UNSAFE",
+      "payload": "test53"
+  }' | grep "session-*" | cut -d' ' -f2)
+
+curl --location '$CATENA_URL/api/v1/accounts' \
+--header "session-id: $SESSION_ID" \
+--header 'Content-Type: application/json' \
+--data '{}'
+
+# {"account":{"id":"account-f378e3c2-8760-4059-ba91-69f5343dfb48","displayName":"test53","authRole":"user","metadata":{},"providers":["PROVIDER_UNSAFE"]}}
+```
+
+[//]: # (TODO: Add the link here.)
+The full postman collection can be found here. Set the http-host environment variable to your domain prior to use.
+
+## Running your game in a multiplayer configuration
+
+Next we will use the server [Sidecar](Sidecar.md) along with our backend to support a multiplayer workflow.
+
+> The sidecar enables you to configure your game to work with one API interface, and deploy it on multiple fleet managers. It can also make requests on behalf of the game server, enabling integrations which require zero server-side integration.
```

Comment:
>  enabling integrations which require zero server-side integration

This is somewhat confusing phrasing.

Reply:
```suggestion
> The sidecar enables you to configure your game to work with one API interface, and deploy it on multiple fleet managers. It can also make requests on behalf of the game server reducing the burden to set up a dedicated game server.
```